keras/backend/tensorflow_backend.py

```diff
@@ -2588,6 +2588,13 @@ def _make_callable(self, feed_arrays, feed_symbols, symbol_vals, session):
         self._symbol_vals = symbol_vals
         self._session = session
 
+    def _numpy_dtype(self, tensor):
```

Comment:
Prefer making it a private utility function rather than a method (we may need it in other places, and if we don't then it would make sense to define it the body of `_call`)